### PR TITLE
README: Fix the plugin name and add badges

### PR DESCRIPTION
**Pull request by oleg-nenashev**
Subj.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
-jenkins-kubernetes-plugin
+Kubernetes plugin for Jenkins
 =========================
 
+[![Jenkins Plugin](https://img.shields.io/jenkins/plugin/v/role-strategy.svg)](https://plugins.jenkins.io/kubernetes)
```
Comment (jglick) [inline README.md]:
```suggestion
[![Jenkins Plugin](https://img.shields.io/jenkins/plugin/v/kubernetes.svg)](https://plugins.jenkins.io/kubernetes)
```

File: README.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
...
+[![GitHub release](https://img.shields.io/github/release/jenkinsci/kubernetes-plugin.svg?label=release)](https://github.com/jenkinsci/kubernetes-plugin/releases/latest)
+[![Jenkins Plugin Installs](https://img.shields.io/jenkins/plugin/i/kubernetes.svg?color=blue)](https://plugins.jenkins.io/kubernetes)
```
Comment (jglick) [inline README.md]: This image seems broken.

Comment (Vlatombe) [reply README.md]: It looks good to me

Comment (oleg-nenashev) [reply README.md]: Works for me. Requests to shields take a while sometimes, try reloading the page

**Pull request merged**